app/src/main/res/layout/main_activity.xml

```diff
@@ -70,6 +77,7 @@
         android:layout_marginTop="-32dp"
         android:layout_marginEnd="20dp"
         android:backgroundTint="@color/black"
+        android:contentDescription="@string/button_that_allows_for_the_sharing_of_media_with_other_apps"
```

Comment:
This name is pretty excessive. It needs to be shorter and camelCase to match the other variables

Reply:
Yeah.  :)